src/kit/InputNumber.tsx

```diff
@@ -21,12 +21,13 @@ const InputNumber: React.FC<InputNumberProps> = forwardRef(
   ({ ...props }: InputNumberProps, ref: React.ForwardedRef<HTMLInputElement>) => {
     const { onFocus, onBlur, inputRef } = useInputNumberEscape(ref);
     const {
-      themeSettings: { className: themeClass },
```

Comment:
I think calling it `themeClass` here is fine, makes it easier to understand the line later where it is combined with `props.className`